CHANGELOG.md

```diff
@@ -9,11 +9,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 
+-   Implement `VoteDataListItem` with Structure and Skeleton module
 -   Implement `ghost` variant for `Button` component
 -   Implement `primaryInverted` variant for `Spinner` component
 
 ### Changed
 
+-   Update `addressUtils` to show first 6 characters with `truncateAddress` method
```

Comment:
I can't see these updates on the `addressUtils` utility

Reply:
Oops I changed it back because it was breaking other tests and ultimately decided it should be in a separate fix ticket.